### PR TITLE
Removes catgirls and purrbation

### PR DESCRIPTION
**Pull request by swissloaf**
### Intent of your Pull Request

catgirls bad

#### Changelog

:cl:  
rscdel: removes felinids and purrbation
/:cl:

### CODE REVIEW

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -1063,8 +1061,6 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 					yogtoggles ^= QUIET_ROUND
 				if("pda")
 					donor_pda = donor_pda % donor_pdas.len + 1
-				if("purrbation")
```
Comment (jcatjmeow) [inline code/modules/client/preferences.dm:1066]: This also removes catboys as well as catgirls. This is clearly an error

**Review by jcatjmeow — CHANGES_REQUESTED** [review]

File: code/modules/client/preferences.dm (hunk 1 above)
Comment (swissloaf) [reply code/modules/client/preferences.dm:1066]: no its not

**Comment by nesisknesis** [comment]
compensate retard

**Comment by swissloaf** [comment]
i just realized half of my commits are missing cause i went onto a separate branch on github desktop :/

**Pull request closed without merging**